src/Storefront/Controller/AddressController.php

```diff
@@ -167,6 +209,9 @@ public function saveAddress(RequestDataBag $data, SalesChannelContext $context,
         );
     }
 
+    /*
+    * @deprecated tag:v6.7.0 - will be removed
```

Comment:
is there an alternative? why is PHPStan not complaining that a `Feature::triggerDeprecationOrThrow` is missing? :sweat_smile: